src/main/scala/com/mozilla/telemetry/streaming/ErrorAggregator.scala

```diff
@@ -20,7 +20,7 @@ import org.joda.time.DateTime
 object ErrorAggregator {
 
   private val allowedDocTypes = List("main", "crash")
-  private val outputPrefix = "error_aggregates/v1"
+  private val outputPrefix = "error_aggregates/v2"
```

Comment:
In that case it depends on the deploy procedure. If we: store the current position in Kafka, write the current aggregates, stop the job, and then start the new job with the same current position, I don't see any reason why the aggregates wouldn't just keep working.